ambry-account/src/main/resources/AccountSchema.ddl

```diff
@@ -51,6 +51,7 @@ CREATE TABLE IF NOT EXISTS Datasets (
     datasetName VARCHAR(235) NOT NULL,
     versionSchema INT NOT NULL,
     retentionCount INT DEFAULT NULL,
+    retentionTimeInSeconds BIGINT NOT NULL,
```

Comment:
Can we add comment, for the usage of each fields?

Reply:
What's retentionCount, userTags?
For lastModifiedTime, do we update it whenever there is a change of the Dataset?

Reply:
I have the comments in dataset.java. for lastModifiedTime, it's same as Account and Container table that any change of the dataset will update the lastModifiedTime.